doc/ref_impl/cntt-ri2/chapters/chapter04.md

```diff
@@ -45,9 +45,9 @@ IDF includes information about network information required by the installer. Al
 
 More details regarding these descriptor files as well as their schema are very well documented in [RI-1 Chapter 8](../../cntt-ri/chapters/chapter08.md#opnfv-descriptor-files-1).
 
-Additionally, modify the BMRA config by first updating `sw_config/bmra/inventory.ini` with the correct servers and functionality planned for each node. For the high availability requirement at least 3 nodes should be running as master with etcd enabled, but only a single master (and worker) is required to deploy the cluster.
```

Comment:
Q: is this removed because it's no longer required? i.e. I wouldn't need to modify that inventory.ini file any more?